Documentation/Contributors/BuildGuide/README.md

```diff
@@ -44,9 +47,26 @@ Cesium ships with a simple HTTP server for testing, run `npm start` after buildi
 
 ```
 npm start
-```
+``` 
+
+Then browse to [http://localhost:8080/](http://localhost:8080/). You should see the following:
+
+* **Apps**
+   * **Cesium Viewer** : our core 3D globe viewer with some basic features included
```

Comment:
This is correct, but I think it's better stated as 'a sample application using asynchronous module definitions (AMD) to create a Cesium 3D globe viewer with some extra features'